spec/classes/profile/haproxy_spec.rb

```diff
@@ -16,6 +16,7 @@
 
       let(:scotch) { { 'ip' => '111.111.111.123', 'hostname' => 'scotch' } }
       let(:soda)   { { 'ip' => '222.222.222.234', 'hostname' => 'soda' } }
+      let(:third_server) { { 'ip' => '333.333.333.345', 'hostname' => 'third_server' } }
```

Comment:
I tried to use Faker::Internet here, like in the keepalive spec, but it kept running twice for reasons I couldn't figure out, so the tests always failed. Eventually, I gave up and gave it static values.